app/rdb/src/i18n/locales/de-DE.yaml

```diff
@@ -105,15 +105,15 @@ modifier.2.value: Haltbarkeit ↑
 modifier.4: Gnadenstoß
 modifier.4.value: Gnadenstoß
 modifier.8: Weitwurf
-modifier.8.value: Throw Range/Speed x {{valueThousandth}} # FIXME
```

Comment:
Can you clean up the FIXME comments? (and other comments)

Reply:
You mean remove them?

Reply:
Yeah remove comments that were basically telling translators what needs to be done

Reply:
Should be cleaned up now